packages/openapi-request-validator/index.ts

```diff
@@ -20,6 +20,7 @@ export interface OpenAPIRequestValidatorArgs {
   requestBody?: OpenAPIV3.RequestBodyObject;
   schemas?: IJsonSchema[];
   componentSchemas?: IJsonSchema[];
+  ajvOptions: Ajv.Options;
```

Comment:
not sure how crazy i am about exposing the options like this.  if we decide to go with a different jsonschema validator in the future, we'd have a breaking change to rename this.  is there another way to pass in the options?  and if all this is just to support nullable, shouldn't we just allow it based on what version of OpenAPI is in use?

Reply:
mm i see, ok it's a good point. Maybe only with the patch did on https://github.com/kogosoftwarellc/open-api/pull/398/files#diff-32b5cc863d07f4a03bd815a69e54f78cR501 are enough to allow the nullable without exposing the options. What do you think?

Reply:
Not sure what you mean, but if it's passing in nullable to ajv when v3 schema is detected,then I'd say yes that would be better